sympy/core/symbol.py

```diff
@@ -220,6 +232,51 @@ class Symbol(AtomicExpr, Boolean):
     >>> bool(A*B*2 == 2*A*B) == True # multiplication by scalars is commutative
     True
 
+    Note that Symbol objects are compared based on their name and assumptions.
+    Two symbols with same name but different assumptions are not equal and
+    treated as distinct in an expression.
+
+    >>> from sympy import Symbol
+    >>> Symbol('x') != Symbol('x', real=True)
+    True
+    >>> eq = Symbol('x') + Symbol('x', real=True); eq
+    x + x
+
+    This ambiguity in naming can be removed using the ``disambiguate`` function.
+
+    >>> from sympy.core.symbol import disambiguate
+    >>> disambiguate(eq)
+    (x + x_1,)
+
+
+    By running `init_printing()` SymPy will automatically pretty-print Greek
+    letters and subscripts.
+
+    >>> from sympy import symbols, init_printing
+    >>> init_printing()
+    >>> alpha, beta, gamma = symbols('alpha beta gamma')
+    >>> alpha + 2 * beta + 3 * gamma #doctest: +SKIP
```

Comment:
Here `init_printing` should be skipped so that it doesn't affect other doctests.